build_and_run.sh

```diff
@@ -21,11 +21,22 @@
 
 set -eu
 
-readonly unzip_dir="${1:-$HOME/Applications}"
+unzip_dir="$HOME/Applications"
+bazel_path="bazel"
+
+while [ $# -gt 0 ]; do
+
+   if [[ $1 == *"--"* ]]; then
```

Comment:
Missed this before, this seems a bit odd (shouldn't need the beginning *, right)?

WDYT about instead using `getopts` with short form args (I guess `-b` for bazel and `-d` for dir) and then you can also have a help option?

Reply:
Sounds good!